app/models/concerns/metadata.rb

```diff
@@ -0,0 +1,29 @@
+module Metadata
+  extend ActiveSupport::Concern
+
+  included do
+    def publisher_metadata
+      metadata = {
+        from: linkable_organisations,
+        first_published: first_published_at,
+        last_updated: updated,
+      }
+
+      unless pending_stats_announcement?
```

Comment:
Can we add a test that covers this? (I think we're not testing when pending_stats_announcements exist)